### PR TITLE
API added SS4 compatibility

### LINKED ISSUE #33: Make module SS4 compatible
As a developer I would like to have this module working with SilverStripe 4

### PR DESCRIPTION
**Pull request by fspringveldt**
Fixes #33

### CODE REVIEW

File: _config.php [hunk 1]
```diff
@@ -1 +1,6 @@
 <?php
+
+namespace SilverStripe\Taxonomy;
+
+
+
```
Comment (robbieaverill) [inline _config.php]: You could just delete this file - it's not needed since we've got things in the `_config` directory

File: composer.json [hunk 2]
```diff
@@ -11,17 +11,24 @@
 		}
 	],
 	"require": {
-		"silverstripe/framework": "^3.1.0"
+		"silverstripe/framework": "^4@dev",
+        "silverstripe/admin": "^1.0"
```
Comment (robbieaverill) [inline composer.json]: Could you mark this as `^1@dev`?

File: _config.php (hunk 1 above)
Comment (fspringveldt) [inline _config.php]: This module in fact doesn't have a _config dir so I've just left it in place.

File: docs/en/developer.md [hunk 3]
```diff
@@ -24,13 +24,7 @@ class TaxonomyTermExtension extends DataExtension
...
-
-```php
-TaxonomyTerm::add_extension('TaxonomyTermExtension');
-```
-
-or with YAML:
+Then add the extension by including this in your `mysite/_config/config.yml`:
 
 ```yaml
 TaxonomyTerm:
```
Comment (robbieaverill) [inline docs/en/developer.md]: These will need to be namespaced now

**Pull request merged**